testing/src/main/kotlin/org/neo4j/connectors/kafka/testing/sink/Neo4jSinkRegistration.kt

```diff
@@ -16,21 +16,25 @@
  */
 package org.neo4j.connectors.kafka.testing.sink
 
+import com.fasterxml.jackson.databind.ObjectMapper
 import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
 import kotlin.time.Duration
 import kotlin.time.Duration.Companion.milliseconds
 import org.neo4j.connectors.kafka.testing.RegistrationSupport
 import org.neo4j.connectors.kafka.testing.RegistrationSupport.randomizedName
 import org.neo4j.connectors.kafka.testing.format.KafkaConverter
 
-internal class Neo4jSinkRegistration(
+class Neo4jSinkRegistration(
     neo4jUri: String,
     neo4jUser: String,
     neo4jPassword: String,
     neo4jDatabase: String,
     retryTimeout: Duration = (-1).milliseconds,
     retryMaxDelay: Duration = 1000.milliseconds,
-    errorTolerance: String = "all",
+    errorTolerance: String = "none",
```

Comment:
do we need that default value if `Neo4jSink` already carries a default value for this?

Reply:
Actually no, we don't need it, I think it was just copy paste from `Neo4jSink` :D

Reply:
Okay, I'm keeping this because we have some sink registration tests which have nothing to do with `errorTolerance`, it's good to keep a default value for sink registration, too.